app/dashboard/views.py

```diff
@@ -899,11 +899,74 @@ def users_fetch_filters(profile_list, skills, bounties_completed, leaderboard_ra
             organizations__contains=[organisation.lower()]
         )
         profile_list = (profile_list1 | profile_list2).distinct()
-
+    if hackathon_id:
+        profile_list = profile_list.filter(
+            hackathon_registration__hackathon=hackathon_id
+        )
     return profile_list
 
 
 
+@require_GET
+def projects_fetch(request):
+    q = clean(request.GET.get('search', ''), strip=True)
+    order_by = clean(request.GET.get('order_by', '-created_on'), strip=True)
```

Comment:
When this gets used, make sure to order by whatever this field says, plus the id as a fallback to prevent duplicates from showing up during pagination: `queryset.order_by(order_by, 'id')`